generators/client/templates/angular/package.json.ejs

```diff
@@ -94,72 +94,72 @@
     "chai-as-promised": "7.1.1",
     "chai-string": "1.5.0",
     <%_ } _%>
-    "codelyzer": "5.2.0",
+    "codelyzer": "5.2.2",
     "copy-webpack-plugin": "5.1.1",
-    "css-loader": "3.3.2",
-    "eslint": "6.7.2",
+    "css-loader": "3.5.3",
+    "eslint": "6.8.0",
     "eslint-config-jhipster": "0.0.1",
-    "eslint-config-prettier": "6.7.0",
-    "eslint-loader": "3.0.3",
-    "file-loader": "5.0.2",
+    "eslint-config-prettier": "6.11.0",
+    "eslint-loader": "4.0.2",
+    "file-loader": "6.0.0",
     "friendly-errors-webpack-plugin": "1.7.0",
     "generator-jhipster": "<%= packagejs.version %>",
-    "html-loader": "0.5.5",
+    "html-loader": "1.1.0",
     "html-webpack-plugin": "3.2.0",
     <%_ if (!skipCommitHook) { _%>
-    "husky": "3.1.0",
+    "husky": "4.2.5",
     <%_ } _%>
-    "jest": "24.9.0",
-    "jest-date-mock": "1.0.7",
+    "jest": "25.4.0",
+    "jest-date-mock": "1.0.8",
     "jest-junit": "10.0.0",
-    "jest-preset-angular": "8.0.0",
+    "jest-preset-angular": "8.1.3",
     "jest-sonar-reporter": "2.0.0",
     <%_ if (!skipCommitHook) { _%>
     <%# Not using the latest version because of https://github.com/jhipster/generator-jhipster/issues/10312 -%>
-    "lint-staged": "8.2.1",
+    "lint-staged": "10.1.7",
```

Comment:
Is it safe now to use latest version? If so the the comment above should be removed

Reply:
Sorry, I missed this one; I'll revert back in a new PR. I cannot test whether this works or not anyways since I don't have a windows machine. :smile:

Reply:
@murdos : Done. :smile: